src/Sinch/Conversation/ConversationRegion.cs

```diff
@@ -5,8 +5,19 @@ namespace Sinch.Conversation
     /// </summary>
     public record ConversationRegion(string Value)
     {
+        /// <summary>
+        ///     US Production
+        /// </summary>
         public static readonly ConversationRegion Us = new("us");
 
+        /// <summary>
+        ///     EU Production
+        /// </summary>
         public static readonly ConversationRegion Eu = new("eu");
+
+        /// <summary>
+        ///     BR Production
+        /// </summary>
+        public static readonly ConversationRegion Br = new("br");
```

Comment:
If a new region is supported by the API, does it require to release a new version of the SDK with the new constant definition?

Reply:
No, there is two ways to provide another value:
* Initilizing same record, not using the constant with `options.ConversationRegion = new ConversationRegion("othernewregion")`
* using `ApiUrlOverrides`